--- a/src/RocketLabs/SellerCenterSdk/Core/Configuration.php
+++ b/src/RocketLabs/SellerCenterSdk/Core/Configuration.php
@@ -67,7 +67,8 @@ class Configuration
      */
     protected function validateUrl($url)
     {
-        $filteredUrl = filter_var($url, FILTER_VALIDATE_URL, FILTER_FLAG_HOST_REQUIRED | FILTER_FLAG_SCHEME_REQUIRED);
+        // FILTER_VALIDATE_URL already validates host and scheme by default
+        $filteredUrl = filter_var($url, FILTER_VALIDATE_URL);

         if (false === $filteredUrl) {
             throw new \InvalidArgumentException(
